keyboards/lyra/keymaps/default/keymap.c

```diff
@@ -0,0 +1,133 @@
+#include QMK_KEYBOARD_H
+#include <stdio.h>
+char wpm_str[10];
+//
+// Keymaps
+//
+enum lyra_layers {
+    /* _M_XYZ = Mac Os, _W_XYZ = Win/Linux */
+    _QWERTY,
+    _FUNCTION
+
+};
+enum custom_keycodes{
+  KC_QWERTY = SAFE_RANGE,
+  KC_FUNCTION
+};
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+    // Layer 0
+    [_QWERTY] = LAYOUT(
+    KC_ESC,		KC_1,		KC_2,		KC_3,				KC_4,		KC_5,		KC_6,				KC_7,		KC_8,		KC_9,		KC_0,		KC_MINS,		KC_EQL,			KC_BSPC,
+    KC_TAB,		KC_LBRC,	KC_Q,		KC_W,				KC_E,		KC_R,		KC_T,				KC_Y,		KC_U,       KC_I,		KC_O,		KC_P,			KC_NUHS,		KC_DEL,			
+    KC_CLCK,	KC_RBRC,	KC_A,		KC_S,				KC_D,		KC_F,		KC_G,				KC_H,		KC_J,       KC_K,		KC_L,		KC_SCLN,		KC_QUOT,		KC_ENT,
+    KC_LSFT,	KC_NUBS,	KC_Z,		KC_X,       		KC_C,       KC_V,       KC_B,            	KC_N,		KC_M,		KC_COMM,	KC_DOT,		KC_SLSH,		KC_UP,			KC_RSFT,
+    KC_LCTL,	KC_LGUI,	KC_LALT,	TT(1),				KC_APP,		KC_PGDN,	KC_SPC,				KC_SPC,		KC_PGUP,	KC_RALT,	KC_RCTL,	KC_LEFT,		KC_DOWN,		KC_RGHT					
+    ),
+
+    // Layer 1
+    [_FUNCTION] = LAYOUT(
+    KC_TRNS,		KC_F1,      KC_F2,      KC_F3,      KC_F4,      KC_F5,		KC_F6,				KC_F7,      KC_F8,      KC_F9,		KC_F10,     KC_F11,			KC_F12,			KC_TRNS,
+    KC_TRNS,		KC_TRNS,    KC_TRNS,	KC_TRNS,    KC_TRNS,    KC_TRNS,	KC_TRNS,			KC_TRNS,    KC_TRNS,    KC_TRNS,	KC_TRNS,    KC_TRNS,		KC_TRNS,		KC_TRNS,
+    KC_TRNS,		KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,	KC_TRNS,			KC_TRNS,	KC_TRNS,	KC_TRNS,	KC_TRNS,	KC_TRNS,		KC_TRNS,		KC_TRNS,
+    KC_TRNS,		KC_TRNS,	KC_TRNS,	KC_TRNS,	KC_TRNS,	KC_TRNS,	KC_TRNS,			KC_TRNS,	KC_TRNS,	KC_TRNS,	KC_TRNS,	KC_TRNS,		KC_TRNS,		KC_TRNS,
+    KC_TRNS,		KC_TRNS,	KC_TRNS,	KC_TRNS,	KC_TRNS,	KC_VOLD,	KC_TRNS,			KC_TRNS,	KC_VOLU,	KC_TRNS,	KC_TRNS,	KC_TRNS,		KC_TRNS,		KC_TRNS	
+    )
+
+};
+
+//
+// Keyboard initialization
+//
+void keyboard_post_init_user(void) {
+    // Wait for keyboard startup
+    wait_ms(100);
+}
+#ifdef OLED_DRIVER_ENABLE
```

Comment:
Ideally, all of this should be moved to the `lyra.c` or `rev1.c` file, instead of here.